src/dec128.rs

```diff
@@ -786,6 +798,48 @@ impl d128 {
     pub fn is_zero(&self) -> bool {
         unsafe { decQuadIsZero(self) != 0 }
     }
+
```

Comment:
Hmm, why? As we can see in libstd, `i16`, `u16`, `i32`, ... all provides `from_*` and `to_*` functions for endianness conversion.